owner/manager.go

```diff
@@ -392,9 +397,10 @@ func (m *ownerManager) SetOwnerOpValue(ctx context.Context, op OpType) error {
 		}
 	})
 
+	leaseOp := clientv3.WithLease(clientv3.LeaseID(m.sessionLease.Load()))
 	resp, err := m.etcdCli.Txn(ctx).
 		If(clientv3.Compare(clientv3.ModRevision(ownerKey), "=", modRevision)).
-		Then(clientv3.OpPut(ownerKey, string(newOwnerVal))).
+		Then(clientv3.OpPut(ownerKey, string(newOwnerVal), leaseOp)).
```

Comment:
when will the owner renew lease?

Reply:
Here https://github.com/pingcap/tidb/pull/43894/files#diff-a0eb33bb4d5cbd8ab32a999bdabe4e3c35dba3e9ac5ad03fa3b8d033334ff8f7R250

Reply:
Will this lead to frequent owner changes?